File: CompilationTest/ExplicitGenericTypeTest/results/ExplicitGenericTypeTest.js
class TestClass {
  testMethod() {
    const result = ["y"];
    const map = {
      x: 5
    };
    const keys = Object.keys(map);
    console.log(result[0]);
    console.log(keys[0]);
  }
}

try {
  new TestClass().testMethod();
} catch(e) {
  console.log(`Exception: ${e.message}`);
}

File: CompilationTest/ExplicitGenericTypeTest/results/ExplicitGenericTypeTest.swift
class TestClass {
  func testMethod() -> Void {
      let result: [String]? = ["y"]
      let map: [String: Int]? = [
        "x": 5
      ]
      let keys: [String]? = Array(map!.keys)
      print(result![0])
      print(keys![0])
  }
}

TestClass().testMethod()